app/services/triannon/ldp_writer.rb

```diff
@@ -167,6 +167,7 @@ def create_base
       g = OA::Graph.new(g)
       g.remove_non_base_statements
       g.make_null_relative_uri_out_of_blank_node
+      g << [RDF::URI.new, RDF.type, RDF::Vocab::LDP.BasicContainer]
```

Comment:
OA::Graph is just a wrapper around RDF::Graph, so you can do any RDF::Graph method on an OA::Graph object.